wagtailio/templates/base.html

```diff
@@ -60,92 +59,74 @@
-            js.src = "https://connect.facebook.net/en_US/sdk.js";
-            fjs.parentNode.insertBefore(js, fjs);
-            }(document, 'script', 'facebook-jssdk'));
-        </script>
-        {% endif %}
-        {% wagtail_ab_testing_script %}
+        {% include "includes/svg_sprites.html" %}
 
         {% wagtailuserbar %}
 
-        <!--[if lt IE 9]>
-            <p class="browserupgrade">You are using an <strong>outdated</strong> browser. Please <a href="http://browsehappy.com/">upgrade your browser</a> to improve your experience.</p>
-        <![endif]-->
-
-        <div class="page">
-            <div data-alert="{% url 'sitewide_alert:sitewide_alert' %}"></div>
-
-            <header class="header">
-                <div class="header__container">
-                    <a class="branding" href="/">
-                        <img class="branding__logo" src="{% static "img/wagtail.svg" %}" alt="Wagtail" />
-                    </a>
-                    <a href="#primary_navigation" class="menu-toggle" aria-label="Toggle menu">
-                        <span></span>
-                    </a>
-                    {% menu_primary calling_page=page %}
-                </div>
-            </header> <!-- /header -->
-
-            {% include "includes/svg_sprites.html" %}
+        <div data-alert="{% url 'sitewide_alert:sitewide_alert' %}"></div>
 
-            {% block sidebar %}{% endblock %}
-
-            <div class="body">
-
-                <section>
-                    {% block content %}{% endblock %}
-                </section>
+        <header class="header">
+            <div class="header__container">
+                <a class="branding" href="/">
+                    <img class="branding__logo" src="{% static "img/wagtail.svg" %}" alt="Wagtail" />
+                </a>
+                <a href="#primary_navigation" class="menu-toggle" aria-label="Toggle menu">
+                    <span></span>
+                </a>
+                {% menu_primary calling_page=page %}
+            </div>
+        </header> <!-- /header -->
 
-                {% block footer %}
-                {% include "includes/footer.html" %}
-                {% endblock %}
+        {% block sidebar %}{% endblock %}
 
-            </div> <!-- /body -->
+        <section>
+            {% block content %}{% endblock %}
+        </section>
 
-            <div class="menu-overlay">
-                {% menu_primary calling_page=page %}
-            </div> {# /menu-overlay #}
-
-        </div> <!-- /page -->
+        {% block footer %}
+            {% include "includes/footer.html" %}
+        {% endblock %}
 
         {% include "includes/cookie_message.html" %}
 
         <a href="https://tally.so/r/wQqagw" target="_blank" class="survey-link">
-            <img src="{% static 'img/clipboard.svg' %}" alt="Help improve wagtail.org clipboard icon">
             <span>Help improve wagtail.org</span>
         </a>
 
+        {% if not DEBUG %}
```

Comment:
Note: I would have thought we'd want this higher on the page, possibly in the `<head/>`

Reply:
Good shout! It sounds like it should go directly after the opening body tag, so i've moved it there. Ref: https://developers.google.com/tag-platform/tag-manager/web